superdesk/macros/set_default_template_metadata.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import superdesk
+from flask_babel import lazy_gettext
+from superdesk import get_resource_service
+
+fields_to_exclude = superdesk.app.config['FIELDS_TO_EXCLUDE']
+fields_to_override = superdesk.app.config['FIELDS_TO_OVERRIDE']
```

Comment:
I would name it more specifically, something like:
`DEFAULT_TEMPLATE_METADATA_MACRO_EXCLUDE`